core/trino-main/src/main/java/io/trino/server/testing/TestingTrinoServer.java

```diff
@@ -262,7 +265,7 @@ private TestingTrinoServer(
         ImmutableMap.Builder<String, String> serverProperties = ImmutableMap.<String, String>builder()
                 .putAll(properties)
                 .put("coordinator", String.valueOf(coordinator))
-                .put("catalog.management", "dynamic")
+                .put("catalog.management", catalogMangerKind.name().toLowerCase(ENGLISH))
```

Comment:
We don't have to convert to `toLowerCase`- We can use `name` as the enum mapping is roubust in Airlift.

Reply:
Done :)